src/main/java/org/folio/rest/utils/ErrorHelper.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.rest.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.folio.rest.jaxrs.model.Error;
+import org.folio.rest.jaxrs.model.Errors;
+import org.folio.rest.jaxrs.model.Parameter;
+import org.folio.rest.tools.RTFConsts;
+
+import javax.ws.rs.core.Response;
+
+public class ErrorHelper {
+
+  private ErrorHelper() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static Errors createValidationErrorMessage(String field, String value, String message) {
+    Errors e = new Errors();
+    Error error = new Error();
+    Parameter p = new Parameter();
+    p.setKey(field);
+    p.setValue(value);
+    error.getParameters().add(p);
+    error.setMessage(message);
+    error.setCode("-1");
+    error.setType(RTFConsts.VALIDATION_FIELD_ERROR);
+    List<Error> l = new ArrayList<>();
+    l.add(error);
+    e.setErrors(l);
+    return e;
+  }
```

Comment:
The method is used only for test cases. This method should be moved to the test package.
It can also be simplified:

  private Errors createValidationErrorMessage(String field, String value, String message) {

    Parameter parameter = new Parameter()
      .withKey(field)
      .withValue(value);

    Error error = new Error()
      .withParameters(Collections.singletonList(parameter))
      .withCode("unique code")
      .withMessage(message)
      .withType(RTFConsts.VALIDATION_FIELD_ERROR);

    return new Errors()
      .withErrors(Collections.singletonList(error));
  }